rules/REQUEST-934-APPLICATION-ATTACK-GENERIC.conf

```diff
@@ -354,6 +354,34 @@ SecRule REQUEST_COOKIES|!REQUEST_COOKIES:/__utm/|REQUEST_COOKIES_NAMES|ARGS_NAME
     setvar:'tx.inbound_anomaly_score_pl2=+%{tx.critical_anomaly_score}'"
 
 
+# [ Generic RCE signatures ]
+#
+# Detects General SSTI attacks.
+# Example: <%= File.open('/etc/passwd').read %> 
+# Note: there is another rule 941380 that checks for {{.*}} regex.
+#
+SecRule REQUEST_COOKIES|!REQUEST_COOKIES:/__utm/|REQUEST_COOKIES_NAMES|ARGS_NAMES|ARGS|XML:/* "@rx ({%.*%}|<%[=]?.*%>)" \
```

Comment:
The capturing group around the entire regex `()` is not necessary. You can remove it.
The `[]` around `=` is not necessary.
I'm not sure if we should make the `.*` lazy with `.*?`. In 941380 I also added the non-greedy option: https://github.com/coreruleset/coreruleset/blob/main/rules/REQUEST-941-APPLICATION-ATTACK-XSS.conf#L1065